app/services/docker/pluginService.js

```diff
@@ -20,30 +20,30 @@ angular.module('portainer.services')
     return deferred.promise;
   };
 
-  service.volumePlugins = function(systemOnly) {
+  servicePlugins = function(systemOnly, Type, Arg) {
```

Comment:
Could you change the signature of the function to `aggregatedPlugins = function(systemOnly, pluginType, pluginVersion)` ?